### ISSUE TITLE
hugin build fails

### ISSUE THREAD

**Issue by sjmudd** [NixOS/nixpkgs#7051]
I'm very new to nix and found it by chance as I was looking to solve a way to install a package that on native centos-6 has rpm dependency hell build problems (centos 6 is rather old).

I thought I could try building it with nix so installed it but had this failure:

```
[me@myhost ~]$ nix-env -i hugin
installing ‘hugin-2013.0.0’
these derivations will be built:
  /nix/store/9z6rd6qxms6i7nkdflh9zvm2791q5kf2-libxmi-1.2.drv
  /nix/store/hqznv6hfaymar7w8fxzqr6i7083wxlnn-enblend-enfuse-4.1.3.drv
  /nix/store/hab9dsds2qlxbnwpqw3sg7p9djxnkqsx-hugin-2013.0.0.drv
building path(s) ‘/nix/store/5grjd003480d57islsillg59ibvgg1ar-libxmi-1.2’
unpacking sources
unpacking source archive /nix/store/vxs163wsycw35dqgcm1yqwqvm3fpf96w-libxmi-1.2.tar.gz
source root is libxmi-1.2
patching sources
configuring
cp: cannot stat '/nix/store/jbms3clffilaic6bvnmsw4jxzympfzav-libtool-2.4.5/share/libtool/config/config.sub': No such file or directory
builder for ‘/nix/store/9z6rd6qxms6i7nkdflh9zvm2791q5kf2-libxmi-1.2.drv’ failed with exit code 1
cannot build derivation ‘/nix/store/hqznv6hfaymar7w8fxzqr6i7083wxlnn-enblend-enfuse-4.1.3.drv’: 1 dependencies couldn't be built
cannot build derivation ‘/nix/store/hab9dsds2qlxbnwpqw3sg7p9djxnkqsx-hugin-2013.0.0.drv’: 1 dependencies couldn't be built
error: build of ‘/nix/store/hab9dsds2qlxbnwpqw3sg7p9djxnkqsx-hugin-2013.0.0.drv’ failed
[me@myhost ~]$ cat .nix-
.nix-channels  .nix-defexpr/  .nix-profile/  
[me@myhost ~]$ cat .nix-channels 
https://nixos.org/channels/nixpkgs-unstable nixpkgs
[me@myhost ~]$ rpm -q centos-release
centos-release-6-5.el6.centos.11.2.x86_64
[me@myhost ~]$ 
```

Right now due to my ignorance with Nix I'm not sure how to fix this but assumed that a package defined was one that built and given the comments about all versions working or something this failure seemed somewhat surprising. I've only tried to install bash (as a test) apart from this package so am not sure what I'm doing wrong.

I notice there's a newer version of hugin available from "upstream", and so would be interested in using that, so if you need me to help get this working (including a newer version) please let me know.

**Comment by domenkozar** [comment]
Did you try `nix-channel --update`? It seems you have outdated channel

**Comment by vcunat** [comment]
This problem was fixed in 0ce3dcd91a. However, the currently newest nixos-unstable channel is a month old and doesn't contain that fix.

**Comment by pSub** [comment]
Closing, as the latest unstable channel does contain the fix.
